packages/calcite-components/conventions/README.md

```diff
@@ -153,6 +153,22 @@ It is recommended to reflect properties that fit the following criteria:
 
 Doing so will give developers more flexibility when querying the DOM. This is important in framework environments where we can't safely assume components will have their attributes set vs properties.
 
+### Property defaults
```

Comment:
In order to not block the fix, can we discuss this separately before making it a convention? There might be some other cases to consider (e.g., [the color-picker ensures the format matches and set default value to hex or hexa depending on alpha being enabled or not](https://github.com/Esri/calcite-components/blob/master/packages/calcite-components/src/components/color-picker/color-picker.tsx#L219-L221)).

Reply:
Yeah lets discuss it. 

As a user, its nice to know what the expected default value for a prop will be. Having these conditional default values does complicate things so we should discuss whether it matters or not.

We may be able to make rules for how these conditionals should be documented since there are other components like this. (such as datepicker `value` where it depends on if the component is a range or not)

We could at least update the doc for colorPicker to document the `default` since it only lists the hex right now. We may also want to enhance our `defaults` test to be able to handle these conditionals.

Reply:
@jcfranco with the colorPicker, if `alphaChannel` is set to `true` after initialization, the value doesn't get updated right? Would a user expect the value to be updated? We should discuss that part as well.

Reply:
Maybe we can make exceptions for props that are mutable? In most cases `value` props are mutable and can be modified by the user and the component.